app/jobs/submit_folio_request_job.rb

```diff
@@ -99,10 +99,14 @@ def pickup_location_id
     end
 
     def get_service_point_code(destination)
-      return destination || Settings.Folio.default_service_point if Settings.ils.bib_model == 'Folio::Instance'
+      # Check if comparable service point code exists, otherwise return default
+      if Settings.ils.bib_model == 'Folio::Instance'
+        code = valid_service_point_code(destination) ? destination : Settings.folio.default_service_point
+      else
+        code = Settings.libraries[destination].folio_pickup_service_point_code
+        code ||= Settings.libraries['GREEN'].folio_pickup_service_point_code
+      end
```

Comment:
```suggestion
      code = if Settings.ils.bib_model == 'Folio::Instance'
        valid_service_point_code?(destination) ? destination : Settings.folio.default_service_point
      else
        Settings.libraries[destination].folio_pickup_service_point_code ||
          Settings.libraries['GREEN'].folio_pickup_service_point_code
      end
```

Reply:
For here, I realized since this is already the FOLIO submit job, I don't need any options for Symphony, so I removed the check altogether